app/models/GmpCalculationResponse.scala

```diff
@@ -16,8 +16,9 @@
 
 package models
 
-import org.joda.time.LocalDate
-import play.api.libs.json.Json
+import java.time.LocalDate
```

Comment:
Make sure joda and java Local Date use same format for backward compatibility

Reply:
Gone through all the code places where this DateTime is used. Execution is fine. But if any uncovered areas are there, they might get exposed during automation testing.
No Action taken for  this.